js/classes/player.js

```diff
@@ -1,13 +1,39 @@
 class Player extends Game {
-  constructor(round, players, gameState, playerNum, playerName) {
+  constructor(round, players, gameState, playerNum, playerName, active) {
     super(round, players, gameState);
     this.playerNum = playerNum;
     this.score = 0;
     this.playerName = playerName;
+    this.active = active || false;
   }
 
-  submitAnswer() {
+  submitAnswer(answer) {
+    let activeQuestion = questionsArray.find((currentQuestion) => {
+      if (currentQuestion.active === true) {
+        return currentQuestion
+      }
+    })
 
+    if (answer.toLowerCase().includes(activeQuestion.answer.toLowerCase())) {
+      newGame.players.find((currentPlayer) => {
+        if (currentPlayer.active) {
+          currentPlayer.score += activeQuestion.pointValue;
+          domUpdates.updateNamesAndScores();
+          currentPlayer.active = false;
+
+        }
+      })
+    } else {
+      newGame.players.find((currentPlayer) => {
+        if (currentPlayer.active) {
+          currentPlayer.score -= activeQuestion.pointValue;
+          domUpdates.updateNamesAndScores();
+          currentPlayer.active = false;
+        }
+      })
+    }
```

Comment:
Again, lotssss of repeat code in this if/else block. The only place where these blocks of code are different is where you increment or decrement the player's score. Tighten up this if/else so that it only wraps that single line of code.